disnake/channel.py

```diff
@@ -2155,6 +2178,7 @@ async def edit(
         position: int = MISSING,
         nsfw: bool = MISSING,
         overwrites: Mapping[Union[Role, Member], PermissionOverwrite] = MISSING,
+        flags: ChannelFlags = MISSING,
```

Comment:
missing in overload

Reply:
https://github.com/DisnakeDev/disnake/pull/642/commits/615e105c3c93a8235c291b158991b1beb124196b